docs/source/03_tutorial/07_set_up_experiment_tracking.md

```diff
@@ -0,0 +1,159 @@
+```
+
+Again, we will need to ensure that the dataset is also specified as an output on `pipeline.py` under the `data_processing` pipeline, as follows:
+
+```python
+node(
+    func=preprocess_companies,
+    inputs="companies",
+    outputs=["preprocessed_companies", "companies_columns"],
+    name="preprocess_companies_node",
+)
+```
+
+Having set up both datasets, we are now ready to generate our first set of experiment tracking data!
+
+## Generating Run data
+
+One of the beauty of native experiment tracking in Kedro is that all tracked data are generated and stored each time you do a Kedro run. Hence, to generat the data, simply do:
+
+```bash
+Kedro run
+```
+
+After the run completes, under `data/09_tracking`, you will now see 2 folders, `companies_column.json` and `metrics.json`. On performing a Kedro run after setting up the tracking datasets, Kedro will generate a folder under the dataset name for each tracked dataset, with each folder containing another folder named under the timestamp of the run. Each subsequent Kedro run will generate a new folder named after the timestamp of the run, in the same directory of the tracked dataset.
+
+You will also see the `session_store.db` generated from your first Kedro run after enabling experiment tracking, which is used to store all the generated run metadata, alongside the tracking dataset, to be used for exposing experiment tracking to Kedro-Viz.
+
+![](../meta/images/experiment-tracking_folder.png)
+
+Try to execute `Kedro run` a few times to generate a larger set of experiment data. You can also play around with setting up different tracking datasets, and check the logged data via the generated JSON data files.
+
+## Accessing Run data and comparing runs on Kedro-Viz
+
+Here comes the fun part of accessing your run data on Kedro-Viz. Having ensured that you are using Kedro-Viz `>=4.1.1` (you can confirm your Kedro-Viz version by running `Kedro info`), run
+
+```bash
+kedro viz
+```
+
+On spinning up Kedro-Viz, you will now see an `experiment tracking` icon on your left. Clicking the icon will bring you to the experiment tracking page (you can also access the page via `http://127.0.0.1:4141/runsList`), where you will now see the set of experiment data generated from your previous Kedro runs, as shown below:
+
+![](../meta/images/experiment-tracking_runsList.png)
+
+You will now be able to access, compare and pin your runs by toggling the `Compare runs` button, as shown below:
+
+![](../meta/images/experiment-tracking_demo.gif)
+
+You can also access a more detailed demo [here](https://kedro-viz-live-demo.hfa4c8ufrmn4u.eu-west-2.cs.amazonlightsail.com/).
+
+s
+
+## Feedback
+
+The Kedro team is constantly working on improving our product, and would love your feedback to help evolve this feature - please reach out to us with any comments or questions via our Discord channel or by via Github issues. (https://github.com/quantumblacklabs/kedro/issues). 
+
+Happy Tracking! :test_tube:
```

Comment:
```suggestion
```

Reply:
You're going to hate me for removing this but it just makes the piece longer than it needs to be 🥺